### PR TITLE
add re.sonny.OhMySVG

### PR DESCRIPTION
**Pull request by sonnyp**
https://github.com/sonnyp/OhMySVG/

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build re.sonny.OhMySVG

**Comment by sonnyp** [comment]
There was a syntax error in the manifest, I have pushed a fix.

**Comment by hfiguiere** [comment]
bot, build re.sonny.OhMySVG

File: re.sonny.OhMySVG.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - "--socket=wayland"
+  - "--require-version=1.1.2"
+modules:
+  - name: OhMySVG
+    buildsystem: meson
+    sources: 
+      "type": "git"
+      "url": "https://github.com/sonnyp/OhMySVG.git"
+      "tag": "v1.0.0"
+      "commit": "c3b505ce98aeec0277a8721b2ea5aa5852e913d9"
```
Comment (hfiguiere) [inline re.sonny.OhMySVG.yaml:18]:
```suggestion
      - type: git
        url: https://github.com/sonnyp/OhMySVG.git
        tag: v1.0.0
        commit: c3b505ce98aeec0277a8721b2ea5aa5852e913d9
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]
this is why it builds (untested though)

File: re.sonny.OhMySVG.yaml (hunk 1 above)
Comment (sonnyp) [reply re.sonny.OhMySVG.yaml]: thanks

could you give it an other shot?

**Comment by hfiguiere** [comment]
bot, build re.sonny.OhMySVG

**Comment by hfiguiere** [comment]
Yu can request it yourself.

**Comment by sonnyp** [comment]
Thanks - everything looks good.

File: re.sonny.OhMySVG.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+app-id: re.sonny.OhMySVG
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: re.sonny.OhMySVG
+finish-args:
+  - "--share=ipc"
```
Comment (barthalion) [inline re.sonny.OhMySVG.yaml:7]:
```suggestion
  - "--share=ipc"
  - "--device=dri"
```

Comment (sonnyp) [reply re.sonny.OhMySVG.yaml:7]: why?

Comment (barthalion) [reply re.sonny.OhMySVG.yaml:7]:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

It probably doesn't matter much in the context of your app but would be nice to fix.

Comment (sonnyp) [reply re.sonny.OhMySVG.yaml:7]: :+1:

**Comment by sonnyp** [comment]
bot, build re.sonny.OhMySVG

**Pull request closed without merging**